spring-cloud-sleuth-brave/src/main/java/org/springframework/cloud/sleuth/brave/bridge/BraveHttpServerRequest.java

```diff
@@ -131,6 +133,10 @@ private boolean resolveFromInetAddress(brave.Span span) {
 					}
 					return span.remoteIpAndPort(addr.getAddress().getHostAddress(), addr.getPort());
 				}
+				else if (delegate instanceof HttpServletRequest) {
+					HttpServletRequest servletRequest = (HttpServletRequest) delegate;
+					return span.remoteIpAndPort(servletRequest.getRemoteAddr(), servletRequest.getRemotePort());
```

Comment:
Just like in the case of `ServerHttpRequest`, could you please do a null-check on `servletRequest.getRemoteAddr()`? As far as I remember, `getRemoteAddr` can return `null` in some cases, I guess that's the reason why there is a null-check above.

Reply:
Done